packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Changed
+- Update getEIP1559Compatibility to match extension ([#1404](https://github.com/MetaMask/core.git/pull/1404))
+- Reset network details when lookupNetwork request fails ([#1403](https://github.com/MetaMask/core.git/pull/1403))
```

Comment:
``` suggestion
Update lookupNetwork to unset `networkDetails.EIPS[1559]` in state instead of setting it `false` if either of its requests for the network ID or network details fails ([#1403](https://github.com/MetaMask/core/pull/1403))